pirlib/cli/dockerize.py

```diff
@@ -90,13 +115,12 @@ def _generate_dockerfile(context_path: pathlib.Path) -> str:
                 "RUN wget https://repo.anaconda.com/miniconda/Miniconda3"
                 "-latest-Linux-$(uname -m).sh -O /tmp/Miniconda3.sh"
             ),
-            f"RUN bash /tmp/Miniconda3.sh -b -p {miniconda3}",
+            f"RUN bash /tmp/Miniconda3.sh -b -p {miniconda3}",  # && rm /tmp/Miniconda3.sh",
```

Comment:
stray comments

Reply:
Let's keep it there as uncommenting it would delete the Miniconda installed shell file, reducing the size of the image.